site/_apps/zuix-hn.txt

```diff
@@ -6,19 +6,19 @@ library: 'ZUIX'
 libraries: 
     - name: 'ZUIX'
 module-bundling: 'zuix-bundler'
-service-worker: 'Application Shell + Workbox'
+service-worker: 'Application Shell + Workbox data caching'
 perfomance-patterns: 'Lazy loaded components with caching'
 server-side-rendering: 'None'
 api: 'Hacker News Firebase API'
 hosting: 'Firebase'
 authors: 
     - name: 'Gene'
 lighthouse: '91/100'
-interactive-em: '5.1s'
-interactive-faster-3g: '4.3s'
-lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170613_Y4_3f81ed03de518aaecc6a8823e74ec401&run=3
-wpt-em-link: https://www.webpagetest.org/result/170613_22_be140a755468b49f0e8480eeb79149ed/
-wpt-faster-3g-link: https://www.webpagetest.org/result/170613_Y4_3f81ed03de518aaecc6a8823e74ec401/
+interactive-em: '4.8s'
+interactive-faster-3g: '3.0s'
+lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170613_KA_4797d0c9289924c6d61abf3326146944&run=1
```

Comment:
New Lighthouse link as current one is down: https://www.webpagetest.org/lighthouse.php?test=170715_NN_e8f85dda8cffc693f00a914dc543bfd2&run=1